packages/react/src/components/Icon/icons/images.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512"><!--! Font Awesome Free 6.6.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc. --><path d="M160 32c-35.3 0-64 28.7-64 64l0 224c0 35.3 28.7 64 64 64l352 0c35.3 0 64-28.7 64-64l0-224c0-35.3-28.7-64-64-64L160 32zM396 138.7l96 144c4.9 7.4 5.4 16.8 1.2 24.6S480.9 320 472 320l-144 0-48 0-80 0c-9.2 0-17.6-5.3-21.6-13.6s-2.9-18.2 2.9-25.4l64-80c4.6-5.7 11.4-9 18.7-9s14.2 3.3 18.7 9l17.3 21.6 56-84C360.5 132 368 128 376 128s15.5 4 20 10.7zM192 128a32 32 0 1 1 64 0 32 32 0 1 1 -64 0zM48 120c0-13.3-10.7-24-24-24S0 106.7 0 120L0 344c0 75.1 60.9 136 136 136l320 0c13.3 0 24-10.7 24-24s-10.7-24-24-24l-320 0c-48.6 0-88-39.4-88-88l0-224z" fill="currentColor" /></svg>
```

Comment:
I _think_ we might be in violation of the license since comments appear to get stripped out, I don't see it anywhere in the resulting dev build or code. 

When you build locally, do you see this license show up anywhere?

Reply:
Looks like this issue has been around. https://github.com/dequelabs/cauldron/pull/654

Reply:
I'm not even sure we necessarily have to keep the license itself in the svg, but it has to be distributed. Let me check on this today and get back to you.

Reply:
They do say "we ask that you do not actively work to remove them from files, especially code." They're asking nicely. 😄

Reply:
> we ask that you do not actively work to remove them from files

I'm not even sure if that's a possibility right now. The default options of svgo preset-defaults should be keeping legal comments per [`removeComments`](https://svgo.dev/docs/plugins/removeComments/) so I'm not quite sure where they are getting stripped.

Reply:
It may be due to this:
> If you aren't using SVGO directly, like through [SVGR](https://github.com/gregberge/svgr), the default plugins may differ from the default preset.

https://svgo.dev/docs/preset-default/

Reply:
After researching this, I have found that since the SVG is rendered as JSX, it's being stripped out because React doesn't render HTML comments.